server/server.js

```diff
@@ -5,8 +5,7 @@ const passport = require('passport');
 const Auth0Strategy = require('passport-auth0');
 const dotenv = require('dotenv');
 const db = require('./db');
-const authRouter = require('./auth');
-const apiRouter = require('./api');
+const apiRouter = require('./api/index');
```

Comment:
if the api directory has an `index.js` file, then it's assumed to use it, so you don't have to be specific here

```suggestion
const apiRouter = require('./api');
```